bots/pr/src/test/java/org/openjdk/skara/bots/pr/ReviewersTests.java

```diff
@@ -42,6 +42,9 @@ public class ReviewersTests {
     private static final String reviewersCommandFinallyOutput = "The total number of required reviews for this PR " +
             "(including the jcheck configuration and the last /reviewers command) is now set to ";
 
+    private static final String reviewProgressTemplate = "Change must be properly reviewed (%d reviews required, with at least %s)";
+    private static final String zeroReviewProgress = "Change must be properly reviewed (no reviews required)";
```

Comment:
Please use uppercase underscore separated identifiers for constants.

```suggestion
    private static final String REVIEW_PROGRESS_TEMPLATE = "Change must be properly reviewed (%d reviews required, with at least %s)";
    private static final String ZERO_REVIEW_PROGRESS = "Change must be properly reviewed (no reviews required)";
```